web/src/components/CreateStorageServiceDialog.tsx

```diff
@@ -185,6 +184,11 @@ const CreateStorageServiceDialog: React.FC<Props> = (props: Props) => {
           Path
           <span className="text-sm text-gray-400 ml-1">(Storage Path)</span>
         </Typography>
+        <Typography className="!mb-1" level="body2">
+          <p className="text-sm text-gray-400 ml-1">{"You can use {year}, {month}, {day}, {hour}, {minute}, {second},"}</p>
+          <p className="text-sm text-gray-400 ml-1">{"{filetype}, {filename}, {timestamp} and any other words."}</p>
+          <p className="text-sm text-gray-400 ml-1">{"e.g., {year}/{month}/{day}/xxx/yyy_{filename}."}</p>
```

Comment:
```suggestion
          <p className="text-sm text-gray-400 ml-1">{"e.g. {year}/{month}/{day}/{filename}.{filetype}"}</p>
```

Reply:
How about change to `{year}/{month}/{day}/path/{filename}.{filetype}`? 

I want to emphasize that we can add any words in the path but not only these placeholder.

Reply:
`path` -> `YOUR_PATH`